README.md

```diff
@@ -242,7 +242,10 @@ $client->segments->getSegment("59c124f770e00fd819b9ce81", ["include_count"=>"tru
 $client->events->create([
     "event_name" => "testing",
     "created_at" => 1391691571,
-    "email" => "test@example.com"
+    "email" => "test@example.com",
+    "metadata" => [
+        "order_date" => 1392036272,
+        "stripe_invoice" => "inv_3434343434" ]
```

Comment:
```suggestion
        "stripe_invoice" => "inv_3434343434"
    ]
```

I would move the closing bracket to the next line to keep it consistent with the rest of the code 😃

Reply:
Yep spot on, thanks!